topics/Dialog-and-popup-sizes.md

```diff
@@ -0,0 +1,278 @@
+
+### How to set the size correctly
+
+1. Select how the content is displayed:
+
+    <table style="header-column">
+        <tr>
+            <td width="40%">
+                Table cell
+            </td>
+            <td>
+                <p>Width: min&nbsp;65&nbsp;px</p>
+                <p>Height for the whole table: min&nbsp;120&nbsp;px</p>
+                <p>Follow guidelines for table width</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Tree
+            </td>
+            <td>
+                Width: min 250&nbsp;px
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Text area
+            </td>
+            <td>
+                <p>Width: min&nbsp;270&nbsp;px, max&nbsp;600&nbsp;px</p>
+                <p>Height: min&nbsp;55&nbsp;px</p>
+                <p>For size and placement follow the <a href="text_area.md" anchor="size-and-placement">text area</a> guidelines</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Diagram
+            </td>
+            <td>
+                A container with horizontal and vertical insets with 100&nbsp;px
+            </td>
+        </tr>
+    </table>
+
+2. What type of content is used:
+
+   <table style="header-column">
+        <tr>
+            <td width="40%">
+                Class/test/file name
+            </td>
+            <td>
+                <p>Width: min&nbsp;250&nbsp;px</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                URL/Path/FQN for class
+            </td>
+            <td>
+                <p>Width: min&nbsp;350&nbsp;px</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Class/test/file name + URL/Path/FQN for class
+            </td>
+            <td>
+                <p>Width: min&nbsp;400&nbsp;px</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Code snippet or editor
+            </td>
+            <td>
+                <p>Width: min&nbsp;400&nbsp;px</p>
+                <p>Height (when multiple lines): min&nbsp;120&nbsp;px</p>
+            </td>
+        </tr>
+        <tr>
+            <td width="40%">
+                Standalone text
+            </td>
+            <td>
+                <p>Width: min&nbsp;300&nbsp;px</p>
+                <p>Height (when multiple lines): min&nbsp;120&nbsp;px</p>
+            </td>
+        </tr>
+    </table>
+
+3. Select the biggest size out of the two to add the minimal size to a component
+
+#### Example 1
+
+The <control>Rename Inheritors</control> dialog has a table with FQNs. Add 350&nbsp;px as the minimum width for a table column. This will make the dialog wider than the recommended 500&nbsp;px but will show more of FQNs.
+
```

Comment:
Possibly show sizes on the image here? Will also help understanding how the correct example differs from the acceptable